bedevere/util.py

```diff
@@ -36,8 +36,9 @@
 
 # Regex pattern to search for tasklist in the issue body
 ISSUE_BODY_TASK_LIST_PATTERN = re.compile(
-    rf"(?P<start>{ISSUE_BODY_OPENING_TAG}\s*```\[tasklist]\s*)"
-    rf"(?P<tasks>.*?)(?P<end>\s*```\s*{ISSUE_BODY_CLOSING_TAG})",
+    rf"(?P<start>{ISSUE_BODY_OPENING_TAG}\s{{1,2}}```\[tasklist\])"
+    rf"(?P<tasks>.*?)"
+    rf"(?P<end>```\s{{1,2}}{ISSUE_BODY_CLOSING_TAG})",
```

Comment:
```suggestion
    rf"(?P<end>```\r?\n{ISSUE_BODY_CLOSING_TAG})",
```